android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/annotation/core/pdf/PdfUtil.kt

```diff
@@ -162,6 +166,38 @@ object PdfUtil {
         )
     }
 
+    private fun getExtractedLinkAnnotation(
+        annotation: PdfDictionary,
+        llx: Float,
+        lly: Float,
+        urx: Float,
+        ury: Float
+    ): Annotation {
+        val xBottomLeftPoint = llx
+        val yBottomLeftPoint = lly
```

Comment:
This is basically renaming the arguments isn't it? If thats the case then just provide meaningful names to the arguments to avoid doing that

Reply:
Done